include/zephyr/ipc/ipc_service.h

```diff
@@ -220,9 +220,9 @@ int ipc_service_open_instance(const struct device *instance);
 
 /** @brief Close an instance
  *
- *  Function to be used to close an instance. All endpoints must be
+ *  Function to be used to close an instance. All bounded endpoints must be
  *  deregistered using ipc_service_deregister_endpoint before this
- *  is called.
+ *  is called (the unused one can be left).
```

Comment:
```suggestion
 *  is called.
```

Reply:
Please add a note about this in `doc/releases/release-notes-3.4.rst`. It is an API change, so better document it.